Patient Registry System/PatientRegistrySystem.Services/UserService.cs

```diff
@@ -0,0 +1,71 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using PatientRegistrySystem.DB.Contexts;
+using PatientRegistrySystem.DB.Entities;
+using PatientRegistrySystem.DB.Repos;
+using PatientRegistrySystem.Domain.Dto;
+using System;
+using System.Linq;
+
+namespace PatientRegistrySystem.Services
+{
+    public class UserService : Controller, IUserService
+    {
+        public IMapper Mapper { get; }
+        public IGenericRepository<User> userRepository { get; }
```

Comment:
these should be `private readonly`.

Reply:
UserRepository should better have an interface IUserRepository that inherits from the generic repo interface. And then you can use here private readonly IUserRepository _userRepository;

Reply:
we will talk later about these interfaces, as we should move them to the domain project.